### ISSUE TITLE
"TypeError: can only join an iterable" in phonemize-lexicon

### ISSUE THREAD

**Issue by mbarnig** [rhasspy/gruut#3]
The script 

`$ zcat <language>_lexicon.txt.gz | gruut <language> phonemize-lexicon`  

generates the following error, for all supported languages:  

`$zcat data/fr-fr_lexicon.txt.gz | gruut fr-fr phonemize-lexicon`   
`Traceback (most recent call last):`   
`  File "/home/mbarnig/.local/bin/gruut", line 8, in <module>`    
`    sys.exit(main())`    
`  File "/home/mbarnig/.local/lib/python3.8/site-packages/gruut/__main__.py", line 90, in main`    
`    args.func(config, args)`   
`  File "/home/mbarnig/.local/lib/python3.8/site-packages/gruut/__main__.py", line 641, in do_phonemize_lexicon`   
`    word_pron_str = "".join(word_pron)`   
`TypeError: can only join an iterable`

**Comment by synesthesiam** [comment]
Thanks! I'll get this fixed shortly.

[This line](https://github.com/rhasspy/gruut/blob/934424a3d05d193575ec4219bc8be36662dacfaa/gruut/__main__.py#L641) needs to be changed to:

```python
            word_pron_str = "".join(word_pron.phonemes)
```

**Comment by mbarnig** [comment]
I changed the line. It works.  Thank you.
But now I get another error where I am lost :   

`............`     
`-der d ə ɹ`   
`-e ə`    
`-e- ə`   
`Traceback (most recent call last):`   
`  File "/home/mbarnig/.local/bin/gruut", line 8, in <module>`  
`    sys.exit(main())`   
`  File "/home/mbarnig/.local/lib/python3.8/site-packages/gruut/__main__.py", line 90, in main`   
`    args.func(config, args)`   
`  File "/home/mbarnig/.local/lib/python3.8/site-packages/gruut/__main__.py", line 663, in do_phonemize_lexicon`   
`    print(word, pron_phonemes_str)`   
`BrokenPipeError: [Errno 32] Broken pipe`

**Comment by mbarnig** [comment]
I used the same command as shown in my first message :   

`zcat <language>_lexicon.txt.gz | gruut <language> phonemize-lexicon`   

In the mean time I have a better understanding of gruut and I tried the following command, without a pipe :   

`python3 -m gruut nl-nl phonemize-lexicon data/nl-nl_lexicon.txt.gz`      

This works as expected for all supported languages.

Example: 

![gruut-phonemize-lexicon-ok](https://user-images.githubusercontent.com/1360633/114566077-35568700-9c72-11eb-8b07-e1f85cede764.jpg)

If I add a pipe, for example 

`python3 -m gruut nl-nl phonemize-lexicon data/nl-nl_lexicon.txt.gz | head -25`  .

the broken pipe error appears after printing a few lines.

I think the problem is not specific gruut related and I close the issue.

Kind regards
